docs/guides/admin/docs/configuration/admin-ui/thumbnails.md

```diff
@@ -40,6 +40,19 @@ The thumbnail preview is automatically downscaled if necessary using the followi
     # Default: editor.thumbnail.preview.downscale
     #thumbnail.preview.profile.downscale=editor.thumbnail.preview.downscale
 
+Since the extraction of a thumbnail can be expensive in case of large files, the update of the thumbnail preview
+image can be configured:
+
+    # Default: partial
+    #thumbnail.preview.mode=partial
+
+If set to `always`, the thumbnail preview image is updated whenever required. If set to `partial`, the thumbnail
+preview image is only updated for uploaded thumbnails to avoid long waiting times that could cause timeouts in
+case of very large files.
+If set to `optimized`, the thumbnail preview image is updated whenever required, but instead of downloading
+files into the workspace (slow for large files), `ffmpeg` is called directly on the Admin node and accesses the
```

Comment:
Needs a big warning about adding ffmpeg to the admin node, since it's normally not installed.